projects/envoyinit/pkg/runner/run.go

```diff
@@ -30,8 +31,17 @@ const (
 func RunEnvoyValidate(ctx context.Context, envoyExecutable, bootstrapConfig string) error {
 	logger := contextutils.LoggerFrom(ctx)
 
-	validateCmd := cmdutils.Command(ctx, envoyExecutable, "--mode", "validate", "--config-yaml", bootstrapConfig, "-l", "critical", "--log-format", "%v")
-	if err := validateCmd.Run(); err != nil {
+	logger.Debugf("starting full envoy validation with size %d", len(bootstrapConfig))
```

Comment:
nit: I find his message of "full envoy validation" a little misleading. The function is invoked from a number of places in the code (waf validation, transformation validation, full envoy validation) and the configuration that we are validating is not always "the full snapshot".

Reply:
yeah i like it. Updated